osvimdriver/openstack/neutron/driver.py

```diff
@@ -19,23 +22,46 @@ def get_network_by_id(self, network_id):
             raise ValueError('network_id must be provided')
         neutron_client = self.__get_neutron_client()
         logger.debug('Retrieving network with id %s', network_id)
-        result = neutron_client.show_network(network_id)
-        return result['network']
+        try:
+            external_request_id = str(uuid.uuid4())
+            driver_request_id  = str(uuid.uuid4())
```

Comment:
driver_request_id is specific to a lifecycle, Please pass this value from the calling function.

Reply:
I have done changes.